### PR TITLE
Revert "commit short message"

### PR DESCRIPTION
**Pull request by gauravsap15**

### CODE REVIEW

**Comment by amohile** [comment]
sdsdsdsd

**Comment by githubteacher** [comment]
Uh oh - this will remove your file - is that what you want to do?

File: mumbai.txt [hunk 1]
```diff
@@ -1,9 +0,0 @@
-things to do in Mumbai, India
-
-Restaunts
--Bade Miyas behind Taj
```
Comment (gauravsap15) [inline mumbai.txt:4]: line comment

**Comment by gauravsap15** [comment]
+1

**Pull request closed without merging**